rocAL_pybind/examples/rocAL_api_video_labels_pipeline.py

```diff
@@ -0,0 +1,181 @@
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+import torch
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import numpy as np
+from parse_config import parse_args
+import os
+
+class ROCALVideoIterator(object):
+    """
+    ROCALVideoIterator for pyTorch.
+
+    Parameters
+    ----------
+    pipelines : list of amd.rocal.pipeline.Pipeline
+                List of pipelines to use
+    size : int
+           Epoch size.
+    """
+
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, display=False ,sequence_length=3):
+
+        try:
+            assert pipelines is not None, "Number of provided pipelines has to be at least 1"
+        except Exception as ex:
+            print(ex)
+
+        self.loader = pipelines
+        self.tensor_format = tensor_layout
+        self.multiplier = multiplier if multiplier else [1.0, 1.0, 1.0]
+        self.offset = offset if offset else [0.0, 0.0, 0.0]
+        self.reverse_channels = reverse_channels
+        self.tensor_dtype = tensor_dtype
+        self.batch_size = self.loader._batch_size
+        self.w = self.loader.getOutputWidth()
+        self.h = self.loader.getOutputHeight()
+        self.n = self.loader.getOutputImageCount()
+        self.rim = self.loader.getRemainingImages()
+        self.display = display
+        self.iter_num = 0
+        self.sequence_length = sequence_length
+        print("____________REMAINING IMAGES____________:", self.rim)
+        color_format = self.loader.getOutputColorFormat()
+        self.p = (1 if color_format is types.GRAY else 3)
+        self.out = np.empty(
+                (self.batch_size*self.n,int(self.h/self.batch_size), self.w,self.p), dtype="ubyte")
+        self.labels = np.zeros(self.batch_size, dtype="int32")
+
+    def next(self):
+        return self.__next__()
+
+    def __next__(self):
+        self.iter_num +=1
+        if(self.loader.isEmpty()):
+            raise StopIteration
+        if self.loader.run() != 0:
+            raise StopIteration
+        #Copy output from buffer to numpy array
+        self.loader.copyImage(self.out)
+        img = torch.from_numpy(self.out)
+        self.loader.getImageLabels(self.labels)     # Get labels
```

Comment:
This function name can be confusing for video. Probably we need to change it to getOutputLabels instead

Reply:
This API is used for other examples and in training iterators. Changing this API name would require change in other files too. Can we add this in this PR or we can make this change in the develop branch